infra/production/base/module/iam/main.tf

```diff
@@ -0,0 +1,67 @@
+data "aws_iam_policy_document" "ecs_node_doc" {
```

Comment:
TLDR: This file is used to define an AWS IAM Role to grant AWS ECS access to our AWS infrastructure.